src/task/script.js

```diff
@@ -389,20 +393,31 @@ function clearUserNotFound() {
 function createSuggestionsList(matches) {
   const listItems = document.getElementById('list-items');
   if (matches.length) {
-    matches.map(({ username }) => {
+    addDisplay(assigneeAvatarLoader);
+    matches.map(({ username, picture = {} }) => {
+      const imageUrl =
+        picture.hasOwnProperty('url') && picture.url
+          ? picture.url
+          : DEFAULT_ASSIGNEE_IMAGE_PATH;
       const listItem = document.createElement('p');
```

Comment:
Creating HTML elements should also move to reusable code. Where it should accept - element name, attributes, class, container.

element name and container should be mandatory. Rest can be optional

Reply:
This needs to be changed in the whole code, so once the code gets restructured, will create a function for this in util.js file and update it in the whole codebase